msu/hooks/skills/skill_container.nut

```diff
@@ -34,6 +34,25 @@
 		}
 	}
 
+	// VANILLAFIX http://battlebrothersgame.com/forums/topic/weapon-skills-lost-on-load-if-multiple-skills-call-unequip-and-equip-in-onadded
+	// if we overwrite add later, we should do this inside the loop in the function
+	local add = o.add;
+	o.add = function( _skill, _order = 0 )
+	{
+		if (!_skill.isStacking())
+		{
+			foreach (skill in this.m.SkillsToAdd)
```

Comment:
Call it `skillToAdd` instead of `skill` for readability

Reply:
`alreadyPresentSkill` is better as `skillToAdd` is confusing with `_skill` which is the skill to be added.